test/e2e/specs/editor/blocks/comments.spec.js

```diff
@@ -8,26 +8,48 @@ const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
  * @typedef {import('@wordpress/e2e-test-utils-playwright').RequestUtils} RequestUtils
  */
 
+/**
+ * Sets a site option, from the options-general admin page.
+ *
+ * This is a temporary solution until we can handle WP options through the REST
+ * API. Note that this function needs to be binded to work properly.
+ *
+ * @param {string} setting The option, used to get the option by id.
+ * @param {string} value   The value to set the option to.
+ *
+ * @return {Promise<string>} A Promise that resolves to the option's previous value.
+ */
+async function _setOption( setting, value ) {
```

Comment:
It's recommended to use the fixture API like in other test files to encapsulate utils in a test suite.

Reply:
Thanks for the tip, I didn't know about the fixture API. 👍